packages/synapse-react-client/src/components/dataaccess/AccessRequirementTable.integration.test.tsx

```diff
@@ -145,21 +162,28 @@ describe('Access Requirement Table tests', () => {
     screen.getByRole('columnheader', {
       name: /^Access Requirement Name/,
     })
+    screen.getByRole('columnheader', { name: 'Type' })
     screen.getByRole('columnheader', { name: 'Related to Projects' })
     screen.getByRole('columnheader', { name: 'Reviewer' })
     screen.getByRole('columnheader', { name: 'Last Modified' })
     screen.getByRole('columnheader', { name: /^Created On/ })
 
     // check the first row of data
-    screen.getByRole('cell', { name: mockSearchResults.results[0].id })
-    screen.getByRole('cell', { name: mockSearchResults.results[0].name })
+    const row = screen.getAllByRole('row')[1]
```

Comment:
Get the first row, then check the cells within that row, so that cells in other rows aren't inadvertently matched when there are multiple cells with the same values (e.g. AR Type).